lib/utilities.js

```diff
@@ -96,9 +96,36 @@ function setStyleProp(style, props) {
  */
 var PRESERVE_CUSTOM_ATTRIBUTES = React.version.split('.')[0] >= 16;
 
+// Taken from
+// https://github.com/facebook/react/blob/cae635054e17a6f107a39d328649137b83f25972/packages/react-dom/src/client/validateDOMNesting.js#L213
+var elementsWithNoTextChildren = new Set([
+  'tr',
+  'tbody',
+  'thead',
+  'tfoot',
+  'colgroup',
+  'table',
+  'head',
+  'html',
+  'frameset',
+  '#document'
```

Comment:
Question: which element would this be?

Reply:
Good question, I'm not sure myself. I just copied it over from react's source. I've removed it since I don't think it is something that part of an HTML structure. [MSDN states](https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeName) that it's an interface to an already loaded document in a browser so it doesn't seem to apply to us.